sdk/core/azure_core/Cargo.toml

```diff
@@ -27,7 +27,11 @@ sha2 = { workspace = true, optional = true }
 tokio = { workspace = true, optional = true }
 tracing.workspace = true
 typespec = { workspace = true, features = ["http", "json"] }
-typespec_client_core = { workspace = true, features = ["derive", "http", "json"] }
+typespec_client_core = { workspace = true, features = [
```

Comment:
Do you have the TOML extension installed? Mine hadn't done this. If you do and it was formatted this way, cool. Just odd since it should be consistent. The TOML extension is one of our recommended ones.

Reply:
Yes I do have the TOML extension on my machine.